code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -1096,7 +1104,7 @@
 				src.updateUsrDialog()
 				return 0
 
-		if (!is_type_in_list(O, blend_items) && !is_type_in_list(O, juice_items))
+		if (!is_type_in_list(O, blend_items) && !is_type_in_list(O, juice_items) && !istype(O,/obj/item/weapon/reagent_containers/food))
```

Comment:
Grinders were still not accepting food, so by forcing it into being accepted, it should be able to grind now.